modules/monitoring-about-monitoring-dashboards.adoc

```diff
@@ -0,0 +1,32 @@
+// Module included in the following assemblies:
+//
+// * observability/monitoring/reviewing-monitoring-dashboards.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="about-monitoring-dashboards_{context}"]
+= About monitoring dashboards
+
+{product-title} provides a set of monitoring dashboards that help you understand the state of cluster components and user-defined workloads.
+
+Use the *Administrator* perspective to access dashboards for the core {product-title} components, including the following items:
+
+* API performance
+* etcd
+* Kubernetes compute resources
+* Kubernetes network resources
+* Prometheus
+* USE method dashboards relating to cluster and node performance
+* Node performance metrics
+
+.Example dashboard in the Administrator perspective
+image::monitoring-dashboard-administrator.png[]
+
+Use the *Developer* perspective to access Kubernetes compute resources dashboards that provide the following application metrics for a selected project:
+
+* CPU usage
+* Memory usage
+* Bandwidth information
+* Packet rate information
+
+.Example dashboard in the Developer perspective
+image::observe-dashboard-developer.png[]
```

Comment:
See SSG guidance on [screenshots](https://redhat-documentation.github.io/supplementary-style-guide/#screenshots). In my opinion, these two **do** add value, but they still need alt text for screen readers. 

I think the main point of including them is to demonstrate the differences, so it might work well to explain the different panels available rather than trying to fully explain the contents.

Reply:
Good idea, thank you for pointing this out! I am not sure I would want to include it in this PR, I will add it as one of the tasks for this additional enhancement PR:  https://issues.redhat.com/browse/OBSDOCS-1527
as that one will be dealing with the screenshots already.

Reply:
Sounds good to me!